src/app.js

```diff
@@ -68,83 +55,14 @@ const create = ({ templatesDir, db, logger }) => {
         ctx.state.logger.log('Handled request');
     });
 
-    const templates = readTemplates(templatesDir);
-    const reportHandlers = createReportHandlers(templates);
-    const reportHandlersWithSuffixes = Object.fromEntries(
-        Object.entries(reportHandlers)
-            .reduce((pv, [key, val]) => [
-                ...pv,
-                [`${key}.csv`, val],
-                [`${key}.xlsx`, val],
-                [`${key}.html`, val],
-            ], []),
-    );
-    const routeHandlers = { ...handlerMap, ...reportHandlersWithSuffixes };
-    logger.push({ routes: Object.keys(routeHandlers) }).log('Serving routes');
-    app.use(router(routeHandlers));
-
-    // Serialise the body to the type we're interested in
-    app.use(async (ctx, next) => {
-        const suffix = ctx.request.path.split('.').pop();
-        switch (suffix) {
-            case 'xlsx': {
-                ctx.state.logger.log('Setting XLSX response');
-
-                const reportName = ctx.request.path.substr(ctx.request.path.lastIndexOf('/'), ctx.request.path.length).replace('/', '').replace('.xlsx', '');
-                const conversion = conversionFactory({
-                    extract: async ({ html, ...restOptions }) => {
-                        const tmpHtmlPath = `/tmp/${reportName}_${Date.now()}.html`;
-
-                        await fs.writeFile(tmpHtmlPath, html);
-
-                        const result = await chromeEval({
-                            ...restOptions,
-                            html: tmpHtmlPath,
-                            scriptFn: conversionFactory.getScriptFn(),
-                        });
+    if (config.oryKetoReadUrl) {
+        app.use(createAuthMiddleware(config.userIdHeader, config.oryKetoReadUrl));
+    }
 
-                        const tables = Array.isArray(result) ? result : [result];
+    const operator = new ReportingOperator(reportData);
+    operator.start();
```

Comment:
It looks like the `.start` method doesn't throw, even if it fails. Which I think would mean the service would appear to start correctly even if the operator is not functioning. Is my analysis correct? Is the behaviour correct?